pixeltable/share/packager.py

```diff
@@ -267,7 +281,8 @@ def restore(self, bundle_path: Path) -> pxt.Table:
                 _logger.info(f'Importing table {tv.name!r}.')
                 self.__import_table(self.tmp_dir, tv, md)
 
-        return replica_tbl
+        with cat.begin_xact(for_write=False):
```

Comment:
I think this fixes a loose end in the packager implementation (I'll confirm when everything is merged) - thanks!